.gitignore

```diff
@@ -2,6 +2,12 @@
 /bin/php-cs-fixer
 /bin/phpspec
 /bin/yaml-lint
+/bin/php-parse
```

Comment:
This is not future-proof: if a new binary is added, we would be forced to add it here as well.
It would be better:
/bin/*
!/bin/poser